src/store/modules/ui.store.js

```diff
@@ -56,7 +56,7 @@ export default {
          *
          * @type Boolean
          */
-        embeddedMode: false,
+        embeddedMode: 'true' === new URLSearchParams(window.location.href).get('embed'),
```

Comment:
Is this really necessary ?

Reply:
I was trying to start the app straight into embed mode, without having the UI flicker between the two states. It's not yet as I wanted it to be, so I'll remove that and create a ticket to tackle this detail